### PR TITLE
Feature: Rebuild and style footer component

### PR DESCRIPTION
**Pull request by waylonturbes**
Co-authored-by: priscila monteiro <pririmonteiro@me.com>
Co-authored-by: moduped <daniemu@mail.uc.edu>
Co-authored-by: Kristian Fulkerson <kristianf451@gmail.com>

## Description

Rebuilding the apps main footer to match the mock-up designs in our Figma file (it's also attached to the associated Trello card).

- Moved footer styles from `global.less` into a new file called `footer.less`
- Cleaned up any unused styles / comments in `global.less`
- Wrapped the entire app with a `Layout` component from Ant Design (see `src/index.js` in "Files changed" tab)
- Broke apart `Footer.js` into two separate files (`MobileFooter.js` & `DesktopFooter.js`) located in the new `src/components/commonn/Footer` directory
- Cleared all unused styles in `footer.less` and added some new styles
- Rebuilt desktop footer to match mock-up from Figma
- Created a new mobile footer based on our Figma design that displays only on screens sizes below `900px`

### Additional Changes

- Fixed proxy error caused by `LoginContainer.js` by removing the `logo` property (it isn't being used) inside the Okta sign-in widget
- Wrapped the `App` component with the `Layout` component from Ant Design so that the header/navbar and footer format well
- Changed `App` so that it returns a `Layout.Content` component instead of a `div`, containing all the routes

## Video Overview

https://youtu.be/lPhAx3QFFGM

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by natecheney-dev — APPROVED** [review]
Looks good, great work with making the footer match what was designed in the figma.

**Review by Cheyenneb96 — APPROVED** [review]
looks great was very neeeded

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ const LoginContainer = () => {
         registration: true,
       },
       // turning this feature on allows your widget to use Okta for user registration
-      logo: 'path-to-your-logo',
       // add your custom logo to your signing/register widget here.
```
Comment (ryan-hamblin) [inline src/components/pages/Login/LoginContainer.js:28]: Removing the property is enough to warrant the removal of the code comment below as well.

File: src/styles/global.less [hunk 2]
```diff
@@ -42,99 +42,18 @@
...
+  font-family: @text__font-family--header;
+  font-weight: 400;
+  letter-spacing: 1px;
 }
 
-.footer-socialIcon {
-  font-size: 2rem;
-}
+//   color:#006C72 !important;
+// }
```
Comment (ryan-hamblin) [inline src/styles/global.less:52]: Remove this please :)

**Review by ryan-hamblin — COMMENTED** [review]
We reviewed this together. Are a few considerations and some minor changes that I'd like to see before final approval.

**Comment by ModupeD** [comment]
This looks so good, good work

**Review by ElchinNasirov — APPROVED** [review]

**Pull request merged**